packages/internal/exported.js

```diff
@@ -0,0 +1 @@
+import './src/types.js';
```

Comment:
As far a I remember, exported.js is an artifact of modules with ambient types and has no benefits for explicitly exported types

Reply:
great, removed